server/graphql/resolvers.ts

```diff
@@ -15,6 +16,11 @@ const resolvers = {
     requestGroup: (_, { id }, { dataSources }): RequestGroupInterface => dataSources.requestGroups.getById(Types.ObjectId(id)),
     requestGroups: (_, __, { dataSources }): Array<RequestGroupInterface> => dataSources.requestGroups.getAll()
   },
+  Mutation: {
+    createRequest: (_, { request }, { dataSources }): Promise<ServerResponseInterface> => dataSources.requests.createRequest(request),
+    updateRequest: (_, { request, id }, { dataSources }): Promise<ServerResponseInterface> => dataSources.requests.updateRequest(request, id),
```

Comment:
The id for the request to update should probably be supplied in the request object itself

Reply:
Done

Reply:
LOL @Anand-Issac i don't know if the changes got pushed for this

Reply:
@meganniu  oops

Reply:
nw :))